basic-spring-boot/build/dev/params

```diff
@@ -0,0 +1,5 @@
+APPLICATION_NAME=spring-rest
+NAMESPACE=basic-spring-boot-dev
+IMAGE_STREAM_TAG_NAME=redhat-openjdk18-openshift:1.1
+SOURCE_REPOSITORY_URL=https://github.com/redhat-cop/spring-rest.git
```

Comment:
Can you move `IMAGE_STREAM_TAG_NAME` and `SOURCE_REPOSITORY_URL` into template as default values?